src/content/self-hosted/install/releases.mdx

```diff
@@ -7,14 +7,22 @@ id: releases
 
 ## 1.15.0
 
-TBD
+22 November 2023
 
 This version is compatible with Kubernetes versions 1.25 to 1.27.
```

Comment:
We need add a removal notice to indicate Kubernetes `1.24` is no longer supported. As we did [here](https://www.okteto.com/docs/self-hosted/install/releases/#removal-notice) for example